docs/gl_objects/merge_request_approvals.rst

```diff
@@ -2,9 +2,49 @@
 Merge request approvals settings
 ################################
 
-Merge request approvals can be defined at the project level or at the merge
+Merge request approvals can be defined at a group level, or a project level or at the merge
```

Comment:
`Merge request approvals can be defined at the group level, or the project level or at the merge`